yelp_package/extra_requirements_yelp.txt

```diff
@@ -1,7 +1,20 @@
 --extra-index-url=https://pypi.yelpcorp.com/simple
+cached-property==1.3.1
 clusterman_metrics==2.2.0
 crypto_lib==4.0.0
+gitdb2==2.0.3
```

Comment:
Are all these other things intentional?

Reply:
slo-transcoder depends on gitpython which depends on gitdb2.

The intent here was to pin all the dependencies of slo-transcoder. Maybe we can get away without doing that?